satellite/satellitedb/overlaycache.go

```diff
@@ -824,13 +817,15 @@ func getNodeStats(dbNode *dbx.Node) *overlay.NodeStats {
 	return nodeStats
 }
 
-func updateRatioVars(newStatus bool, successCount, totalCount int64) (int64, int64, float64) {
+func updateReputation(newStatus bool, α, β, λ, w float64, totalCount int64) (int64, float64, float64) {
```

Comment:
It's really cool having Greek letters in the code, but let's stick to ASCII characters. We can have `alpha`, `beta`, `lambda` and `omega` instead of `α, β, λ, w`. I am even not sure if the last one is the Greek "omega" or the English "double u".